src/RingGrids/grids/full_clenshaw.jl

```diff
@@ -0,0 +1,38 @@
+struct FullClenshawArray{T, N, ArrayType <: AbstractArray{T, N}} <: AbstractFullGridArray{T, N, ArrayType}
```

Comment:
Add docstring for the N-dim types as well? (Same comment applies to all other grid types)